deploy/kubedirector/crd-app.yaml

```diff
@@ -130,6 +132,8 @@ spec:
                         items:
                           type: string
                           minLength: 1
+                          maxLength: 15
+                          pattern: '[a-z0-9]([-a-z0-9]*[a-z0-9])?'
```

Comment:
Might need the begin/end markers in this pattern, like with the other pattern you added above.